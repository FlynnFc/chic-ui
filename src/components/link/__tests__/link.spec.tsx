import Link from '..';
import { render } from 'test/utils';

describe('Link Component', () => {
  it('Renders Correctly', () => {
    const LinkGroup = render(
      <>
        <Link href="https://github.com/karishmashuklaa/chic-ui">
          Link to Chic-UI Repository
        </Link>
        <Link href="https://github.com/karishmashuklaa/chic-ui" disabled>
          Disabled Link to Chic-UI Repository
        </Link>
      </>
    );
    expect(LinkGroup).toMatchSnapshot();
  });
});
