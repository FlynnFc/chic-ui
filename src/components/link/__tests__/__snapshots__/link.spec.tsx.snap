// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link Component Renders Correctly 1`] = `
Array [
  <a
    className="sc-bdfBQB hfJzcz"
    disabled={false}
    href="https://github.com/karishmashuklaa/chic-ui"
  >
    Link to Chic-UI Repository
  </a>,
  <p
    className="sc-bdfBQB kihIlz"
    disabled={true}
    href="https://github.com/karishmashuklaa/chic-ui"
  >
    Disabled Link to Chic-UI Repository
  </p>,
]
`;
